index.html

```diff
@@ -741,6 +738,40 @@
         </section>
 
         <section>
+          <h2 id='ldpcvget'>HTTP GET</h2>
+          <p>
+            An <a>LDPCv</a> MUST respond to <code>GET Accept: application/link-format</code> as
+            indicated in [[!RFC7089]]
+            <a href='https://tools.ietf.org/html/rfc7089#section-5'>section 5</a> and specified in
+            [[!RFC6690]] <a href='https://tools.ietf.org/html/rfc6690#section-7.3'>section 7.3</a>.
+          </p>
+          <p>
+            An implementation SHOULD respond with the <code>Allow</code> header as outlined in 
+            <a href='#ldpcvoptions'></a>.
+          </p>
+          <p>
+            An implementation MUST respond with a <code>Content-Type</code> header.
+          </p>
+          <p>
+            If an <a>LDPCv</a> supports <code>POST</code>, then it MUST respond with 
+            <code>Accept-Post</code>. If <code>POST</code> is supported but bodies are not
+            accepted, then the <a>LDPCv</a> MUST respond with <code>Accept-Post: */*; p=0.0</code>.
+          </p>
+          <p>
+            An implementation MUST respond with <code>Accept-Patch</code> if <code>PATCH</code>
+            requests to the <a>LDPCv</a> are allowed.
+          </p>
+        </section>
+        
+        <section>
+          <h2 id='ldpcvhead'>HTTP HEAD</h2>
+          <p>
+            The <code>HEAD</code> method is identical to <code>GET</code> except that the server MUST
```

Comment:
Seems redundant.

Reply:
I was mirroring the way that HEAD is defined section 3.6.  LDPCvs are fairly distinct from other LDPRs so I figured it would make sense to spell expectations for them in similar detail.  I don't feel strongly about it though, so if it doesn't feel like this adds anything I can go ahead and remove it